features/journalist_can_create_article.feature

```diff
@@ -0,0 +1,20 @@
+@javascript
+
+Feature: Create articles inside category
+    As a journalist,
+    in order to share the latest news,
+    I would like to able to create an article and choose the right category for it.
+
+  Background:
+    Given the following categories exist
+    | name     |
+    | Politics |
+    When I visit the create page
+
+  Scenario: Journalist create article inside category
+    When I fill in "Header" with "Learning Rails 5"
+    And I fill in "Subheader" with "Excited about learning a new framework"
+    And I fill in "Byline" with "Angelica the Great"
+    And I fill in "Body" with "Angelica in craftacademy"
+    And I select 'Politics' 
+    Then I click "Submit"
```

Comment:
You should probably test for the notice and maybe if it's actually saved... like:

```
And("there should be a Article with header {string} in the database") do |expected_header|
  expect(Article.last.header).to eq expected_header
end 
```

Reply:
Not sure whether in this situation it is best practice to put it back or leave it as it is.